esmvaltool/diag_scripts/land_carbon_cycle/diag_zonal_correlation.py

```diff
@@ -421,30 +421,27 @@ def main(diag_config):
         " and 2d in Carvalhais et al. (2014).", ['corr', 'perc'], ['zonal'],
         _get_ancestor_files(diag_config, 'tau_ctotal'))
 
-    if diag_config['write_netcdf']:
```

Comment:
@koir-su Do you agree with removing the possibility to disable writing NetCDF output? If you would like to keep this, it should be documented in the recipe documentation and the option needs to be moved to the recipe. See #2005 for more information.

Reply:
Since the options are effectively on their way out (#2005), I suggest we go ahead. @koir-su, if you prefer to have them here, please open a new issue to add them as individual options for this recipe.